uint8 JNTS_OUT_OF_SYNCH = 0
uint8 SYS_READY = 1
uint8 RUNNING = 2
uint8 MONITOR_BREAK = 3
uint8 STOP_JOINTS = 4
uint8 RESTORE_SAFE = 5
uint8 UNINDEXED = 6
uint8 sys_status
tm_cab/TOPIC_STATUS j1_stat
tm_cab/TOPIC_STATUS j2_stat
tm_cab/TOPIC_STATUS j3_stat